impeller/toolkit/interop/paint.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_IMPELLER_TOOLKIT_INTEROP_PAINT_H_
+#define FLUTTER_IMPELLER_TOOLKIT_INTEROP_PAINT_H_
+
+#include "flutter/display_list/dl_paint.h"
+#include "impeller/geometry/color.h"
+#include "impeller/toolkit/interop/color_filter.h"
+#include "impeller/toolkit/interop/color_source.h"
+#include "impeller/toolkit/interop/formats.h"
+#include "impeller/toolkit/interop/image_filter.h"
+#include "impeller/toolkit/interop/impeller.h"
+#include "impeller/toolkit/interop/mask_filter.h"
+#include "impeller/toolkit/interop/object.h"
+
+namespace impeller::interop {
+
+class Paint final
+    : public Object<Paint, IMPELLER_INTERNAL_HANDLE_NAME(ImpellerPaint)> {
+ public:
+  Paint();
+
+  ~Paint() override;
+
+  Paint(const Paint&) = delete;
+
+  Paint& operator=(const Paint&) = delete;
+
+  const flutter::DlPaint& GetPaint() const;
+
+  void SetColor(Color color);
+
+  void SetInvertColors(bool invert);
```

Comment:
Lets leave this one off. Its kind of a weird one off exposed via flutter before we had color filter matrix. I'd like to remove it at some point in the future.

Reply:
Wilco